test-runner-frontend/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "homepage": "https://adarsh-jaiswal.github.io/graphql-java-test-runner",
+  "name": "graphql-java-test-runner",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/material": "^5.8.7",
+    "@mui/x-data-grid": "^5.12.3",
+    "@ramonak/react-progress-bar": "^5.0.1",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
it looks like we can move some of those dependencies into `devDependencies`

Reply:
Done, also I deleted some dependencies we are not using.